doc/dbus/bus/org.opensuse.Agama.Language1.bus.xml

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
```

Comment:
Would it be possible to actually start the service and run the introspection? I'm a bit afraid that this static data can easily get out of sync with the code... :thinking:

Reply:
Right.
(I'll Resolve this as it is the same thing as your comment on the README)